### ISSUE TITLE
Dataset problem

### ISSUE THREAD

**Issue by liuliuliu1111** [hahnec/hahnec#1]
Hello! RF-ULM is an awesome work! It’s very helpful that you made the project open source, but I met some problems when I tried to run the code on my computer, I'd like to know if you can help me. 
When I run the code, it shows that: **ModuleNotFoundError: No module named 'datasets.pala_dataset.pala_iq'**.The readme file mentioned that the data are at https://doi.org/10.5281/zenodo.7883227 or https://doi.org/10.5281/zenodo.4343435, but I don’t know what the dataset has to do with PalaDatasetIq and PalaDatasetRf. Could you please tell me how to use the data in your code? 
A screenshot of the problem is attached below. Looking forward to your reply! Thank you!
![image](https://github.com/hahnec/hahnec/assets/30686614/e74f3ef7-9c70-4191-a2df-0f582e3ded54)

**Comment by hahnec** [comment]
Yes, that's simply the dataloader. I currently have limited time, but can release an updated version later. Note that the dataloader is not the main contribution of this work. When using other data than stated here, one would need to replace this part.

**Comment by hahnec** [comment]
Meanwhile, a reference implementation for dataloading can be found at https://github.com/AChavignon/PALA, which was used in this project.
